access/types/stdasync.py

```diff
@@ -336,7 +335,7 @@ def _requireActions(exercise):
     Checks that some actions are set.
     '''
     if "actions" not in exercise or len(exercise["actions"]) == 0:
-        if not settings.CONTAINER_MODE or "container" not in exercise:
+        if not "container" in exercise:
```

Comment:
We don't need the `_requireActions` function at all anymore since we don't use the `actions` field in the exercise config.yaml at all anymore.

Reply:
Done!